<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Processing</class>
 <widget class="QWidget" name="Processing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>900</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DECODE</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(4, 15, 15);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="Titles">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="Title">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>70</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 210, 105);</string>
       </property>
       <property name="text">
        <string>DECODE</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Link">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
         <bold>true</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;tg://resolve?domain=XcoderBS&quot;&gt;&lt;span style=&quot; font-size:14pt; text-decoration: underline; color:#007af4;&quot;&gt;https://t.me/XcoderBS&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="Selection1" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTextEdit" name="Input1">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;
background-color: rgba(255, 255,255, 60);
border: 2px solid rgba(252, 255, 255, 80);
border-radius: 10px;</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
        </property>
        <property name="documentTitle">
         <string/>
        </property>
        <property name="lineWrapMode">
         <enum>QTextEdit::LineWrapMode::WidgetWidth</enum>
        </property>
        <property name="markdown">
         <string/>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:15pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="placeholderText">
         <string>INPUT FILE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Select1">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>30</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton{
background-color: rgba(255, 255,255, 60);
border: 2px solid rgba(252, 255, 255, 80);
border-radius: 10px;
color: white;}
QPushButton:hover:!pressed{
 background-color: rgba(255, 255,255, 80);
 border: 2px solid rgba(252, 255, 255, 210);
}
QPushButton:pressed{
background-color: rgba(255, 255,255, 100);
border: 2px solid rgba(252, 255, 255, 220);}</string>
        </property>
        <property name="text">
         <string>SELECT</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="Selection2" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTextEdit" name="Input2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;
background-color: rgba(255, 255,255, 60);
border: 2px solid rgba(252, 255, 255, 80);
border-radius: 10px;</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
        </property>
        <property name="documentTitle">
         <string/>
        </property>
        <property name="markdown">
         <string/>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:15pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="placeholderText">
         <string>OUT FOLDER (OPTIONALLY)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Select2">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>30</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton{
background-color: rgba(255, 255,255, 60);
border: 2px solid rgba(252, 255, 255, 80);
border-radius: 10px;
color: white;}
QPushButton:hover:!pressed{
 background-color: rgba(255, 255,255, 80);
 border: 2px solid rgba(252, 255, 255, 210);
}
QPushButton:pressed{
background-color: rgba(255, 255,255, 100);
border: 2px solid rgba(252, 255, 255, 220);}</string>
        </property>
        <property name="text">
         <string>SELECT</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
border: 2px solid rgba(252, 255, 255, 80);
border-radius: 10px;</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>518</width>
        <height>374</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>1111111</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="Output">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>11111</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255,255, 60); color:white;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="text">
          <string>CONSOLE:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextInteractionFlag::LinksAccessibleByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="Run">
     <property name="font">
      <font>
       <pointsize>50</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{
background-color: rgba(255, 255,255, 60);
border: 2px solid rgba(252, 255, 255, 80);
border-radius: 10px;
color: white;}
QPushButton:hover:!pressed{
 background-color: rgba(255, 255,255, 80);
 border: 2px solid rgba(252, 255, 255, 210);
}
QPushButton:pressed{
background-color: rgba(255, 255,255, 100);
border: 2px solid rgba(252, 255, 255, 220);}</string>
     </property>
     <property name="text">
      <string>DECODE</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="ExitBtn">
     <property name="font">
      <font>
       <pointsize>50</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{
background-color: rgba(255, 255,255, 60);
border: 2px solid rgba(252, 255, 255, 80);
border-radius: 10px;
color: white;}
QPushButton:hover:!pressed{
 background-color: rgba(255, 255,255, 80);
 border: 2px solid rgba(252, 255, 255, 210);
}
QPushButton:pressed{
background-color: rgba(255, 255,255, 100);
border: 2px solid rgba(252, 255, 255, 220);}</string>
     </property>
     <property name="text">
      <string>CLOSE</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Input1</tabstop>
  <tabstop>Run</tabstop>
  <tabstop>Input2</tabstop>
  <tabstop>Select1</tabstop>
  <tabstop>Select2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
